jekyll/_cci2/deploy/configure-deploy-markers.adoc

```diff
@@ -21,20 +21,209 @@ Deploy markers provide a lightweight way to log your deployments without requiri
+** The `component-name` parameter sets the name that will be displayed in the CircleCI UI.
+** The `namespace` parameter is optional and can be provided to use a value other than `default`.
 
-Add a `log release` step at the end of your _deployment_ job, for example:
+If you are deploying multiple components or to multiple environments from a single workflow, you will have to provide the command with a deployment name.
+This identifies the deployment your are planning so that you can later on reference it to update its status.
+This can be done as shown below:
+
+[,yml]
+----
+jobs:
+  deploy-my-service:
+    executor: some-executor
+    steps:
+      - run: circleci run release plan <deploy-name> --environment-name=<some-environment-name> --component-name<=>some-component-name> --target-version=<some-version-name>
+----
+
+In the example the positional argument `deploy-name` is the arbitrary value that will be used to identify the deployment and should be unique within the workflow.
+If not specified the deployment name will be set to `default`.
+
+=== 1.2. Update the deploy status
+
+After deploying your application, you can update the status of the deployment to `RUNNING` by running the `circleci run release update` command in a new step.
+
+[,yml]
+----
+circleci run release update --status=running
+----
+
+If you are deploying multiple components or to multiple environments from a single workflow, you will have to provide the command with a deployment name.
+This value should match the value you provided when you planned the deploy.
+
+[,yml]
+----
+circleci run release update <deploy-name> --status=running
+----
+
+Now you can add `on_success` and `on_failure` steps at the end of your deployment job, to handle the final status update of the deploy.
+
+pass:[<!-- vale off -->]
+[,yml]
+----
+- run:
+    name: Update planned release to SUCCESS
+    command: |
+      circleci run release update \
+        --status=SUCCESS  
+    when: on_success 
+- run:
+    name: Update planned release to FAILED
+    command: |
+      if [ -f failure_reason.env ]; then
+        source failure_reason.env
+      fi
+      circleci run release update \
+        --status=FAILED \
+        --failure-reason="$FAILURE_REASON"
+    when: on_fail 
+----
+pass:[<!-- vale on -->]
+
+This will update the status of the deploy to `SUCCESS` or `FAILED` depending on the outcome of your job.
+In this example, the `failure_reason.env` file can be created by a previous step in the job. This can be done, for example, in a step in which we are validating the status of the deployment.
+You can do that as shown below:
+
+[,yml]
+----
+echo "FAILURE_REASON='Deployment was not found'" > failure_reason.env
+----
+
+=== 1.3 Update the deploy status to canceled
+
+If you want to update your deployment to canceled when the deploy job is canceled, you can do so by adding the following job to your configuration.
+
+[,yml]
+----
+cancel-deploy:
```

Comment:
I think it's helpful for readers if you expand examples like this to the top level yaml key (that might not be the right way to phrase that but...) so include `jobs:`